terminal/fancy-bash-prompt.sh

```diff
@@ -192,9 +192,9 @@ prompt_command_hook()
 	local background_input="none"
 	local texteffect_input="bold"
 
-	local separator_char=$'\uE0B0'
```

Comment:
It returns `'|>'` once I install it, has single quote on there. I don't know why :D, that's why I hard code it. Maybe you have a better solution about it.